markliederbach/finna.xml

```diff
@@ -23,21 +22,7 @@
         </Publish>
     </Networking>
     <WebUI>http://[IP]:[PORT:8784]/</WebUI>
-    <Environment>
-        <Variable>
-            <Name>BASE_URL</Name>
-        </Variable>
-        <Variable>
-            <Name>LOG_LEVEL</Name>
-        </Variable>
-    </Environment>
-    <Labels />
```

Comment:
These are not directly referenced in the [DockerTemplateSchema](https://wiki.unraid.net/DockerTemplateSchema) spec, so I'm removing them.